### ISSUE TITLE
Domain function // Extra characters in the address bar

### ISSUE THREAD

**Issue by asupreme1337** [louislam/uptime-kuma#1563]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/uptime-kuma/security/policy)

### 📝 Describe your problem

Good afternoon. I set up the status page, but I can't figure out the function of domains. I tried adding another domain, setting up CNAME and nginx, but it doesn't give the result I want to achieve.
Now my page has a domain "_status.example.ru_", it takes me to the main status page, but it has the form in the address bar of the type: "_status.example.ru/status/example_".

**How do I get rid of extra characters in the address bar (/status/example)?**

I looked through several other topics on my question, but I didn't find anything detailed. I want my main and only status page to be displayed immediately when navigating through the domain and without extra characters in the address bar.
Users have a lot of questions for this reason, it would be cool if someone wrote a special instruction about this, so that new questions do not appear.

Thanks.

### 🐻 Uptime-Kuma Version

latest

### 💻 Operating System and Arch

Ubuntu 20.04

### 🌐 Browser

Yandex Browser

### 🟩 NodeJS Version

17.9.0

**Comment by louislam** [comment]
Please read: https://github.com/louislam/uptime-kuma/wiki/Status-Page#domain-names-for-status-pages

**Comment by asupreme1337** [comment]
Do I understand correctly that my status page should have the address "**default**" so that nothing superfluous is displayed?

**Comment by asupreme1337** [comment]
Yes, I've seen it, but it's still a bit unclear. The main part of my nginx configuration looks like this:

```
server_name status.example.ru uptime.example.ru;
location @fallback {
proxy_pass http://127.0.0.1:3001;
proxy_redirect http://127.0.0.1:3001 /;
proxy_set_header X-Forwarded-Host $host;
proxy_set_header   X-Real-IP $remote_addr;
proxy_set_header   X-Forwarded-For $proxy_add_x_forwarded_for;
proxy_http_version 1.1;
proxy_set_header   Upgrade $http_upgrade;
proxy_set_header   Connection "upgrade";
access_log off;
}
```
	
The whole site is located on the main domain - status, and "uptime" is set for the status page "default". But the page still opens as "/status/default".

**Comment by louislam** [comment]
You can also try this.
https://github.com/louislam/uptime-kuma/issues/1555#issuecomment-1107924839

**Comment by asupreme1337** [comment]
Wow, thank you very much! Damn nginx. Now everything works as it should.
